src/clr-angular/modal/modal.html

```diff
@@ -15,7 +15,7 @@
          role="dialog"
          [attr.aria-hidden]="!_open"
          [attr.aria-labelledby]="modalId">
-
+      <div class="clr-screen-reader-only">{{commonStrings.modalContentStart}}</div>
```

Comment:
Using `*-sr-only` is fairly standard in place of the longer `*-screen-reader-only`.

Are you open to changing this to `clr-sr-only`?

Reply:
Can do.